fiftyone/utils/eval/coco.py

```diff
@@ -251,6 +253,7 @@ class COCODetectionResults(DetectionResults):
         precision: an array of precision values of shape
             ``num_iou_threshs x num_classes x num_recall``
         recall: an array of recall values
+        recall_sweep: an array of recall values of shape ``num_iou x num_classes``
```

Comment:
You can't add new positional arguments to `EvaluationResults` subclasses because existing evaluations won't have this value and thus `dataset.load_evaluation_results("<existing_eval_key>")` would start failing.

Reply:
I made this optional and the panel doesn't throw any errors now.